client/src/components/classesPage/forms/ListForm.js

```diff
@@ -224,10 +224,24 @@ function ListForm(props) {
   };
 
   const handleCheckBox = () => {
+    const googleProfile = JSON.parse(localStorage.getItem('profile'));
+    const teacherInfo = {
```

Comment:
Nice job using JSON parse, I need to use this in the authentication.js logic, I think my code there could've been cleaner if I just parsed the profile object. Still learning things from you every day!